packages/core/src/abstract/ReceivedInteraction.ts

```diff
@@ -0,0 +1,97 @@
+import { ComponentType, CrossBuild, InteractionRawOptions, LogLevel } from "../index.js"
+import { GeneratedMessage } from "@crossbuild/types"
+
+export interface ReceivedInteractionData {
+	id: string
+	key: string
+	source: string
+	type: ComponentType
+	original: unknown
+	server: {
+		id: string
+		ownerId?: string
+		name?: string
+		iconURL?: string
+		description?: string
+	} | null
+	channel:
+		| ({
+				id: string
```

Comment:
what in the formatting

Reply:
Shows like this on vscode, thats how prettier/esbuild did it
<img width="564" alt="CleanShot 2023-08-11 at 19 08 28@2x" src="https://github.com/crossbuildjs/crossbuild/assets/35580099/23d9f59c-44d5-4e19-a04c-c6e5c01ff780">